rust-peer/src/main.rs

```diff
@@ -179,7 +213,7 @@ fn create_swarm() -> Result<Swarm<Behaviour>> {
 
     // Create a Kademlia behaviour.
     let mut cfg = KademliaConfig::default();
-    cfg.set_query_timeout(Duration::from_secs(5 * 60));
```

Comment:
@p-shahi was it a deliberate choice to set `query_timeout` to 5 minutes? This pull request removes the setting. Fine by you?

Reply:
I believe it was added when we were debugging message propagation issues. Thanks for removing it!